src/app/containers/ErrorMain/index.stories.jsx

```diff
@@ -4,6 +4,7 @@ import { ServiceContextProvider } from '#contexts/ServiceContext';
 import ErrorMain from '.';
 
 storiesOf('Pages|Error Page', module)
+  .addParameters({ chromatic: { disable: true } })
```

Comment:
I think we should continue to test our error pages.

Reply:
Removed this from ErrorMain